lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -202,3 +203,53 @@ def test_report_stores_results(self, mock_time):
 
                     self.assertEqual(return_val, UPDATE_STATUS_SUCCEEDED)
                     mock_store_rows.assert_called_once_with(self.course.id, filename, [test_header] + test_rows)
+
+
+class TestInstructorCourseForumsReport(TestReportMixin, InstructorTaskCourseTestCase):
+    """
+    Tests that course forums response report generation works.
+    """
+    def setUp(self):
+        self.course = CourseFactory.create(org=TEST_COURSE_ORG,
+                                           number=TEST_COURSE_NUMBER,
+                                           display_name=TEST_COURSE_NAME)
+
+        self.current_task = Mock()
+        self.current_task.update_state = Mock()
+
+    def tearDown(self):
```

Comment:
Since you are subclassing TestReportMixin, this is redundant: https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor_task/tests/test_base.py#L243